### PR TITLE
[CONS BLOG] Spatial Filtering w-Dask & Spatialpandas

### PR DESCRIPTION
**Pull request by bskinn**
## Text styling

- [ ] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [ ] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

**Comment by bskinn** [comment]
@ericdatakelly Yes, that's part of the migration task. All of the posts tagged with "needs: changes" have had the post bodies only partly migrated.

File: apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+**Each benchmarking test follows these steps:**
+
+Preprocess data (if necessary) one time
+1. Calculate geohash or Hilbert curve 
+2. Spatially sort data
+3. Save data in partitions
+
+Filter data for each query
+1. Select points from dataset that are within 1, 10, 100, 1000, 10000 random zip code polygons distributed around the contiguous US
```
Comment (ericdatakelly) [inline apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md:123]:
```suggestion
4. Select points from dataset that are within 1, 10, 100, 1000, 10000 random zip code polygons distributed around the contiguous US
```

File: apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md [hunk 2]
```diff
@@ -217,13 +217,15 @@ Other approaches were considered and experimented with. AWS Athena is a serverle
 
 ## Are you working with large datasets?
 
-Reach out to Quansight for a free consultation by sending an email to [connect@quansight.com](email).
+Reach out to Quansight for a free consultation by sending an email to [connect@quansight.com][email].
```
Comment (bskinn) [inline apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md:220]: The website Markdown builder autodetects this as an email and formats it as a `mailto:` link without any markup at all, actually.

```suggestion
Reach out to Quansight for a free consultation by sending an email to connect@quansight.com.
```

File: apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+
+[lexicographically]: https://en.wikipedia.org/wiki/Lexicographic_order
+
+[scipy2020 repo]: https://github.com/Quansight/scipy2020_spatial_algorithms_at_scale
+
+[gps points]: https://planet.openstreetmap.org/gps/simple-gps-points-120604.csv.xz
+
+[polygons]: (https://www2.census.gov/geo/tiger/TIGER2019/ZCTA5/tl_2019_us_zcta510.zip)
+
+[email]: mailto:connect@quansight.com
```
Comment (bskinn) [inline apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md:242]:
```suggestion
```

File: apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+
```
Comment (bskinn) [inline apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md:243]:
```suggestion
```

**Review by bskinn — COMMENTED** [review]
Narrow review just of the `connect@quansight.com` email address.

File: apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+## _Machine Specifications_
+
+Although a cloud cluster was used in production, the benchmark results presented here were performed on a workstation. The specs are included below:
+- Processor: AMD Ryzen Threadripper 2970WX 24-Core Processor
+- RAM: 64 GB
+- For this comparison Dask Workers were limited to:
+- 4 Workers
+- 2 Threads per Worker
+- 3 GB RAM per Worker
```
Comment (ericdatakelly) [inline apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md:137]: Indent the last three bullets
```suggestion
  - 4 Workers
  - 2 Threads per Worker
  - 3 GB RAM per Worker
```

**Comment by ericdatakelly** [comment]
@bskinn This is ready for review.  

Need to make an issue:
- [ ] Convert tables to markdown

**Comment by bskinn** [comment]
@ericdatakelly:

> Convert tables to markdown

I don't think most of these tables can be represented in Markdown. The Markdown table spec is pretty limited.

If we want these rendered as HTML tables instead of images, I think we're looking at custom React work. Punting this until after the migration.

**Pull request merged**